gravitee-exchange-controller/gravitee-exchange-controller-core/src/main/java/io/gravitee/exchange/controller/core/cluster/ControllerClusterManager.java

```diff
@@ -99,6 +120,42 @@ private void handleClusteredReply(Message<ClusteredReply<?>> clusteredReplyMessa
         }
     }
 
+    private void handleChannelsRebalancing() {
```

Comment:
Few questions:

- Could it have consequences if N pods are scaled at the same time?
- How do we know that, when a member is detected, the pod is ready to accept incoming traffic (so the rebalancing has chances to reach the newly scaled pod)?

Behind these questions, I'm wondering if we should we run the rebalance periodically instead and just set a boolean rebalance to true when a new member is detected?